exporters/prometheus/exporter_test.go

```diff
@@ -672,3 +676,164 @@ func TestDuplicateMetrics(t *testing.T) {
 		})
 	}
 }
+
+func TestConcurrentCollect(t *testing.T) {
```

Comment:
```suggestion
func TestCollectConcurrentSafe(t *testing.T) {
```

(this pattern is widely used)